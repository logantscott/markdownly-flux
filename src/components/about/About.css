.About {
  padding-left: 10px;
  display: flex;
  img {
    width: 200px;
    border-radius: 100px;
    border: 1px solid #333;
  }
  figure {
    width: 200px;
    border: 2px solid black;
    border-radius: 15px;
    padding: 10px;
    background: #999;
  }
  figcaption {
    text-align: center;
  }
}

.name {
  font-weight: bold;
}