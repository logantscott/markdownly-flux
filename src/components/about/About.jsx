import React from 'react';
import styles from './About.css';
import Developer from '../developers/Developers';

const About = () => {

  return (
    <main className={styles.About}>
      <Developer />
    </main>
  );
};

export default About;
