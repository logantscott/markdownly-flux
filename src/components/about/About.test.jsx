import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

describe('About component', () => {
  it('renders About', () => {

    const wrapper = shallow(
      <About />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
