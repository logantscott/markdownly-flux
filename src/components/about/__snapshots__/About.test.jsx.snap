// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders About 1`] = `
<main
  className="About"
>
  <Developers />
</main>
`;
